reports/html.go

```diff
@@ -17,34 +17,51 @@ func sanatizeHTML(dirty error) string {
 }
 
 type htmlTemplateInput struct {
-	Reports []Report
-	Tests   map[string][]string
+	Reports    []Report
+	Web5Tests  map[string][]string
+	TbDEXTests map[string][]string
 }
 
 func WriteHTML(reports []Report, filename string) error {
 	slog.Info("writing html report", "reports", len(reports))
 
 	testmap := make(map[string]map[string]bool)
+	tbdexTestMap := make(map[string]map[string]bool)
 	for _, report := range reports {
 		for category, tests := range report.Results {
 			if _, ok := tests[category]; !ok {
-				testmap[category] = map[string]bool{}
+				if report.SDK.Type == "web5" {
+					testmap[category] = map[string]bool{}
```

Comment:
nit: could we rename this to `web5TestMap`?

Reply:
you shouldn't need to initialize like this..., but if it breaks pls use `make(map[string]bool)`